Tests.App/Services/MutagenControllerTest.cs

```diff
@@ -37,6 +37,15 @@ public void CreateTempDir()
         TestContext.Out.WriteLine($"temp directory: {_tempDirectory}");
     }
 
+    [TearDown]
+    public void DeleteTempDir()
```

Comment:
Do we really want to do this? Makes it harder to debug. I assumed the OS would clear the directories out for us eventually, like on restart

Reply:
They get persisted unfortunately. I'll make it so it only deletes if the test passes